shared/i18n/locales/en_US/translation.json

```diff
@@ -40,6 +40,7 @@
   "Dark": "Dark",
   "Light": "Light",
   "System": "System",
+  "Appearance": "Appearance",
```

Comment:
I have no idea what this is lol

Reply:
auto-generated – it should be on master already but nbd